tests/test-helpers/src/lib.rs

```diff
@@ -20,11 +25,59 @@ use rand::{
     Rng,
     RngCore,
 };
+use serde_json::Value;
 
 pub mod builder;
 pub mod counter_contract;
 pub mod fuel_core_driver;
 
+pub fn predicate() -> Vec<u8> {
+    vec![op::ret(1)].into_iter().collect::<Vec<u8>>()
+}
+
+pub fn get_graphql_extension_field_value(result_json: &str, name: &str) -> u64 {
```

Comment:
`fuel-core-client` should fetch this information in the same way how it is done for the required block height. We have `ExtensionsResponse` where you can add several new fields.

Implementation details: In the case of the CPV and STFV we don't need any policy, we just can fetch the latest one and provide a getter

Reply:
Updated in ef9c44522aa804204730a18ac827b2280d5ec587